src/web_socket_adapter_text.ts

```diff
@@ -0,0 +1,96 @@
+import { WebSocketAdapter, SocketCloseHandler, SocketErrorHandler, SocketMessageHandler, SocketOpenHandler } from './web_socket_adapter'
+
+import { b64EncodeUnicode, b64DecodeUnicode } from "./utils";
+import { ApiNotification } from './api.gen';
+
+export class WebSocketAdapterText implements WebSocketAdapter {
```

Comment:
I think the default socket adapter should be merged into the interface type file.